src/views/Home/components/MarketData.tsx

```diff
@@ -45,16 +45,6 @@ const MarketData: React.FC = () => {
     })
   }
 
-  const handleCGIButton = () => {
-    setIndexSelector(1)
-    setProductMetaData({
-      name: productTokensBySymbol.CGI.name,
-      symbol: productTokensBySymbol.CGI.symbol,
-      image: productTokensBySymbol.CGI.image,
-      url: '/cgi',
-    })
-  }
-
   const handleEthFliButton = () => {
     setIndexSelector(2)
```

Comment:
Looks like these values need to be updated, otherwise selecting `BTC2x-FLI` on the home page chart breaks.
Once you do that, will need to update the button selector logic too

```
        <Button
          full
          size={'sm'}
          text='BTC2x-FLI'
          variant={indexSelector === 3 ? 'default' : 'secondary'}
          onClick={handleBtcFliButton}
        />
        ```

Reply:
done